index.js

```diff
@@ -6,7 +6,8 @@ const typeMapping = {
   KAFKA: 'kafka',
   POSTGRESQL: 'postgresql',
   REDIS: 'redis',
-  MONGODB: 'mongodb'
+  MONGODB: 'mongodb',
+  AMQP: 'amqp'
```

Comment:
I better understand why you were talking about rabbitmq support here. I see the service binding specifies it's type as "rabbitmq" so using the version in this PR does not work as there is no match with amqp as the type, it seems wrong to me that the binding operator reports the type of back end versus the type of protocol to be used, however, I'll have to think more on that one.

Reply:
created this issue to asked the service binding operator team how this should work - https://github.com/redhat-developer/service-binding-operator/issues/1110

Reply:
in index.js adding this at the top

```JavaScript
const aliases = {
  amqp: ['rabbitmq']
}

and then changing the check for the bindingType to

```JavaScript
        if (bindingType === typeMapping[type] || aliases[typeMapping[type]].includes(bindingType)) {
```
seems to work around the issue.

```

Reply:
going to add that as commit so that I can test further.